CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # 3.1.0
 
+
+## AutoDisabling form
+
+Form autoDisables after submit when the prop `autoDisable` is set to true. The props `afterFormValidation` and `onSubmit` are passed a `enableForm` callback function which can be used to reactivate the form.
+
+### Example Code
+  ``` 
+  <Form
+    autoDisable
+    onSubmit={ this.saveContact }
+    cancel={ false }>
```

Comment:
do you need cancel false in your example? also the `>` would normally fall onto the next line

```
<Form
  onSubmit={ this.saveContact }
  autoDisable
>
  { children }
</Form>
```